webscraping/node/course-info/storer.js

```diff
@@ -10,44 +10,44 @@
 /**
  * imports
  */
-var fs = require('fs');
-var assert = require('assert');
-var MongoClient = require('mongodb').MongoClient;
-var argv = require('minimist')(process.argv.slice(2));
+let fs = require('fs');
+let assert = require('assert');
+let MongoClient = require('mongodb').MongoClient;
+let argv = require('minimist')(process.argv.slice(2));
 
 /**
  * constants
  */
 const DIR = '../../r/course-info-jsonfiles/';
 const jsonFilenameRegex = /_document.json/;//filter by filename
-var mongoServerUrl = 'mongodb://138.197.6.26:27017/';
-var devDbName = "courseplannerdb-dev";
-var prodDbName = "courseplannerdb";
-var dbName = (argv.prod) ? prodDbName : devDbName;
-var dbFullUrl = mongoServerUrl + dbName;
+let mongoServerUrl = 'mongodb://138.197.6.26:27017/';
+let devDbName = "courseplannerdb-dev";
+let prodDbName = "courseplannerdb";
+let dbName = (argv.prod) ? prodDbName : devDbName;
+let dbFullUrl = mongoServerUrl + dbName;
 
```

Comment:
This thread of conversation was closed in a phone call.  To summarize:
I was asking because I did not like the idea of `scrapeTheWeb.sh` ever putting data directly into the prod db.  I wanted it to only do the dev db, and then we could manually validate the dev db contents and then manually copy the devdb over to proddb when we felt like.  I was concerned that we might accidentally scrape bad data into our prod db.
However, given the automatic validation implemented in this pull request (as well as nobody else sharing my reluctance to scrape right into the prod db), we have agreed that my concerns were unfounded.
Merging.